person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative './nameable'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] Kindly make sure that the program is working properly, currently when I try to run the app, the following error appears:
![image](https://user-images.githubusercontent.com/62290545/210415839-10612744-bc09-4f2f-92f6-9795ebb26aa6.png)

Reply:
Thanks 🙏🙏 @ShadyShawkat I didn't see that error, I'll find a solution and implement it.